### ISSUE TITLE
Emoji are small again

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#26401]
### Steps to reproduce

1. Look at a message with emoji, either in the composer or the timeline

#### What did you expect?

Emoji should have a larger font size than the surrounding text

#### What happened instead?

Emoji are given the same font size as the surrounding text

### Operating system

NixOS unstable

### Browser information

Firefox 118.0.1

### URL for webapp

develop.element.io

### Application version

Element version: f1a39b3e4d37-react-eb7ce666b748-js-4bcea2cead97 Olm version: 3.2.14

### Homeserver

Not relevant

### Will you send logs?

No

**Comment by t3chguy** [comment]
Regressed by https://github.com/matrix-org/matrix-react-sdk/pull/11757

**Comment by Johennes** [comment]
Adding this to the A11y/Compound board view. @kerryarchibald maybe you could give this a look on the back of the PR mentioned above?

**Comment by kerryarchibald** [comment]
Reverting https://github.com/matrix-org/matrix-react-sdk/pull/11774 and reopening https://github.com/vector-im/element-web/issues/26326

Should have tested more carefully, I thought inline emojis being the same size as the text looked correct. Percy tests would have caught this.
